espresso/streamer.go

```diff
@@ -131,23 +131,23 @@ func (s *EspressoStreamer[B]) Reset() {
 
 // Handle both L1 reorgs and batcher restarts by updating our state in case it is
 // not consistent with what's on the L1. Returns true if the state was updated.
-func (s *EspressoStreamer[B]) Refresh(ctx context.Context, finalizedL1 eth.L1BlockRef, safeBatchNumber uint64, safeL1Origin eth.BlockID) (bool, error) {
```

Comment:
This bool was needed only to reset the batcher state every time Refresh updated the block height. AFAIU it was needed in early development as a workaround for some bug in new batcher loops, but now it seems unnecessary and breaks AltDA.

Reply:
Makes sense. Corresponding redundant comment remove in e919fac2f.